### ISSUE TITLE
rename command needs to be reworked

### ISSUE THREAD

**Issue by joshsedl** [webksde/ddev-vscode-devcontainer-drupal-template#29]
### Rename command should also change Folder
Something like this:

- mv route/to/ddev route/to/ddevNew
- change root to ddevNew, otherwise:
![Unbenannt](https://user-images.githubusercontent.com/37101709/136187821-e7da158e-bde4-4d4f-9950-b65867f5e7fb.PNG)

### Rename has to reinstall the site again

- see drowl-init

### Install Modules

-see drowl-init

### Some VSCode Extensions might be added again

-Intelliphense

### Basically drowl-init on rename!

**Comment by joshsedl** [comment]
No need for a rename command anymore, as a project will be dynamically named after the project folder name
